tests/actions/test_log_event.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
+#  Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#  Authors: Daniel Chaves <daniel.chaves@ridgerun.com>
+#           Marisol Zeledon <marisol.zeledon@ridgerun.com>
+
+import unittest
+
+import csv
+from rr.actions.log_event import LogEvent, LogEventError, log, validate_csv, set_file_headers, find_max_probability, log
```

Comment:
normally, you treat the unit as a black box and test through the public api. It's uncommon to test private methods directly, just a tip for the next tests.